frontend/src/app/supportAdmin/ManageFacility/FacilitySelectFilter.tsx

```diff
@@ -58,40 +64,47 @@ const FacilitySelectFilter: React.FC<FacilitySelectFilterProps> = ({
         className="bg-base-lightest padding-left-3 padding-right-3 padding-bottom-1"
       >
         <div className="grid-row grid-gap padding-bottom-2 flex-align-end">
-          <div className="desktop:grid-col-4 tablet:grid-col-4 mobile:grid-col-1">
-            <Dropdown
-              label="Organization"
-              options={[
-                {
-                  label: "- Select -",
-                  value: "",
-                },
-                ...organizationOptions,
-              ]}
-              onChange={onSelectOrg}
-              selectedValue={manageFacilityState.orgId}
+          <div
+            data-testid={"org-selection-container"}
+            className="desktop:grid-col-4 tablet:grid-col-4 mobile:grid-col-1 margin-top-1em"
+          >
+            <Label htmlFor="manage-facility-org-select">
+              Organization <Required />
+            </Label>
+            <ComboBox
+              name={"Organization"}
+              id={"manage-facility-org-select"}
+              options={organizationOptions}
+              onChange={(val) => {
+                onSelectOrg(val);
+              }}
               disabled={loading}
-              required={true}
```

Comment:
The required true was needed to pass a11y scan. Does the combobox handle the required in a different way?

Reply:
@BobanL do you remember how you found the issue [6408](https://github.com/CDCgov/prime-simplereport/issues/6408)?. Want to make sure we don't need the required at the input level before resolving this combo! :D

Reply:
This happens when you test this as a form. I ran through it and it looks like it's still being flagged. I think for the purpose of accessibility we might need to figure out how to add [required](https://www.deque.com/blog/anatomy-of-accessible-forms-required-form-fields/#:~:text=While%20all%20screen,can%20perceive%20it.).
![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/83192b99-956c-404e-9afd-0416850ceeb6)

Reply:
oh actually Truss gives us an escape hatch for this. Will push a fix soon

Reply:
<img width="521" alt="Screenshot 2023-09-06 at 4 35 30 PM" src="https://github.com/CDCgov/prime-simplereport/assets/29645040/5251cfe1-9b0c-4fbc-b48a-397ec396e3e9">

The latest commit adds to prop but if someone could run the scan for me again would appreciate it!

Reply:
Tested it with axe and didn't find any problems!

Reply:
When I ran it on the latest changes I saw this error but am ignoring because I believe this is a false positive jsyk
<img width="1226" alt="Screenshot 2023-09-07 at 09 42 39" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/532107aa-4c44-47fc-b1f3-da98393dbd4e">